proposed/bandwidth-considerations.md

```diff
@@ -0,0 +1,80 @@
+_Note: This recommendation is a DRAFT for consideration of the MODX Governance Advisory Board._
+
+
+To be mindful of the cost users pay to access the MODX Next management experience the core shall effectively leverage the browser cache.
+
+
+**Editor: JP de Vries**
+**First publish draft: TBD**
+
+
+## Goals of Recommendation
+
+
+To make the MODX Next management experience more accessible and less expensive for our global user base. To optimize the page load times of the MODX Next manager across the web as a whole. To promote stability of the MODX Next manager experience. To add costly features and enhancements in a prgoressive manor so they can be opted out&nbsp;of.
+
+
+## Relevant Recommendations
+As bandwidth considerations is part of accessibility best practices, [DRAFT] Accessibility Mandate. This recommendation is proposed as a guideline accepted by the MODX Advisory Board and enforced by the Accessibility Mandate.
+
+
+## Recommendation
+Leverage the browser cache and stability of the progressively enhanced manager with the following practices:
+
+
+Front End (CSS, JS) assets delivered with the core shall not be compressed or concatenated on the server side.
```

Comment:
This should probably be clarified to mean that they are not compressed or concat'd _dynamically_?